src/phoenix/config.py

```diff
@@ -7,6 +7,7 @@
 ENV_PHOENIX_PORT = "PHOENIX_PORT"
 ENV_PHOENIX_HOST = "PHOENIX_HOST"
 ENV_NOTEBOOK_ENV = "PHOENIX_NOTEBOOK_ENV"
+ENV_TRACE_ENDPOINT = "PHOENIX_TRACE_ENDPOINT"
```

Comment:
keep it generic?
```suggestion
ENV_RECEIVER_ENDPOINT = "PHOENIX_RECEIVER_ENDPOINT"
```

Reply:
Oh yeah evals. Was copying OTEL naming but maybe I'll just call it collector